optuna/samplers/_tpe/multi_objective_sampler.py

```diff
@@ -33,16 +33,12 @@ class MOTPESampler(TPESampler):
         consider_prior:
             Enhance the stability of Parzen estimator by imposing a Gaussian prior when
             :obj:`True`. The prior is only effective if the sampling distribution is
-            either :class:`~optuna.distributions.UniformDistribution`,
```

Comment:
The same discussion as optuna/multi_objective/samplers/_motpe.py here.

Reply:
Done by keeping the changes from https://github.com/optuna/optuna/pull/3181!